ASM/c/triforce.c

```diff
@@ -90,14 +90,12 @@ void draw_triforce_count(z64_disp_buf_t* db) {
     gDPSetCombineMode(db->p++, G_CC_MODULATEIA_PRIM, G_CC_MODULATEIA_PRIM);
     gDPSetPrimColor(db->p++, 0, 0, 0xDA, 0xD3, 0x0B, alpha);
 
-    text_print(text , draw_x, draw_y_text);
+    text_print(db, text , draw_x, draw_y_text);
```

Comment:
This code was already formatted this way but might as well fix it.

```suggestion
    text_print(db, text, draw_x, draw_y_text);
```